webirc/kustomization.yaml

```diff
@@ -7,6 +7,12 @@ resources:
 replicas:
 - name: webirc
   count: 2
+configMapGenerator:
+- name: kiwiirc-config
+  options:
+    disableNameSuffixHash: true
+  files:
+  - kiwiirc-config.json
```

Comment:
could you put this in a `files/` subdirectory?

Should it actually be called `kiwiirc-config.json`? or just `config.json` ?